### DISCUSSION TITLE
Can't connect to my account anymore

### DISCUSSION THREAD

**Discussion by Wenzo96** [community/community#120065] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello I come to the community because I can't seem to connect to my account anymore, I have the email/username and password but I don't have TOTP or 2FA. I had to reset my computer a few months ago because because it was slow and out of space so I figured getting it to factory setting would be easier to have it all new and working, but before doing the reset I forgot to transfer all the codes and keys that were on a document file. I do have all the proof that it is my account, like all the emails, my vercel account is still connected to my github, but I just can t seem to connect to my account. I checked all the app on my phone and I don t have any authentication app so I m not entirely on how I setup the 2FA in the first place. Any help would be appreciated, I have 2 years worth of projects on it and loosing all of them would be a big setback, if anyone can provide some help on how can I have access to it again would be amazing. Thanks !

**Comment by ebndev** [comment]
Hi @Wenzo96 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
